portfolio/src/main/webapp/blogs.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <!-- Navigation Bar-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <div class="topnav" id="myTopnav">
+    <a href="/index.html">Home</a>
+    <a href="/blogs.html" class="active">Blog</a>
+    <a href="/projects.html">Projects</a>
+    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
```

Comment:
What does this do? I don't see myFunction defined in your commits. Am I just missing it?

Reply:
Same question for the other places where it appears.